Description: Use "ps2write" instead of "pswrite" Ghostscript output device in the "pdftops" CUPS filter. The "pswrite" turns the text characters into bitmaps and this causes several problems. Also add the "-dNOINTERPOLATE" option to the Ghostscript command line as it improves Ghostscript's performance vastly.
Author: till.kamppeter@gmail.com
Bug-Ubuntu: https://launchpad.net/bugs/361772
Bug-Ubuntu: https://launchpad.net/bugs/362186
Bug-Ubuntu: https://launchpad.net/bugs/369503
Bug-Ubuntu: https://launchpad.net/bugs/668800

--- a/filter/pdftops.c
+++ b/filter/pdftops.c
@@ -270,9 +270,10 @@
   pdf_argv[2] = (char *)"-dNOPAUSE";
   pdf_argv[3] = (char *)"-dBATCH";
   pdf_argv[4] = (char *)"-dSAFER";
-  pdf_argv[5] = (char *)"-sDEVICE=pswrite";
-  pdf_argv[6] = (char *)"-sOUTPUTFILE=%stdout";
-  pdf_argc    = 7;
+  pdf_argv[5] = (char *)"-dNOINTERPOLATE";
+  pdf_argv[6] = (char *)"-sDEVICE=ps2write";
+  pdf_argv[7] = (char *)"-sOUTPUTFILE=%stdout";
+  pdf_argc    = 8;
 #endif /* HAVE_PDFTOPS */
 
   if (ppd)
